visivo/logging/logger.py

```diff
@@ -29,7 +29,7 @@ def set_type(self, type: TypeEnum):
         else:
             self.spinner = Halo(text="Loading", spinner="dots")
             self.spinner.start()
-            time.sleep(0.1)
+            time.sleep(0.1)  # do we still need this?
```

Comment:
Oh wow. Maybe not. I was trying to figure out an issue I don't even remember now.

Reply:
Nice! Gonna pull it out then. I wonder if this is impact our import speed 🤔